File: csharp/MapleStory.OpenAPI/Src/Dto/Character/CharacterBeautyEquipmentSkinDTO.cs
using Newtonsoft.Json;

namespace MapleStory.OpenAPI.Dto
{
    /// <summary>
    /// 캐릭터 피부 정보
    /// </summary>
    public class CharacterBeautyEquipmentSkinDTO
    {
        /// <summary>
        /// 피부 명
        /// </summary>
        [JsonProperty("skin_name")]
        public string SkinName { get; set; }

        /// <summary>
        /// 색상 계열
        /// </summary>
        [JsonProperty("color_style")]
        public string? ColorStyle { get; set; }

        /// <summary>
        /// 피부 색조
        /// </summary>
        [JsonProperty("hue")]
        public int? Hue { get; set; }

        /// <summary>
        /// 피부 채도
        /// </summary>
        [JsonProperty("saturation")]
        public int? Saturation { get; set; }

        /// <summary>
        /// 피부 명도
        /// </summary>
        [JsonProperty("brightness")]
        public int? Brightness { get; set; }
    }
}

File: java/src/main/java/dev/spiralmoon/maplestory/api/tms/dto/character/CharacterBeautyEquipmentSkinDTO.java
package dev.spiralmoon.maplestory.api.tms.dto.character;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

/**
 * 角色膚色資訊
 */
@AllArgsConstructor
@NoArgsConstructor
@Data
@ToString
public class CharacterBeautyEquipmentSkinDTO implements dev.spiralmoon.maplestory.api.common.dto.character.CharacterBeautyEquipmentSkinDTO {

    /**
     * 膚色名稱
     */
    @JsonProperty("skin_name")
    private String skinName;

    /**
     * 色系
     */
    @JsonProperty("color_style")
    private String colorStyle;

    /**
     * 膚色色調
     */
    @JsonProperty("hue")
    private Integer hue;

    /**
     * 膚色彩度
     */
    @JsonProperty("saturation")
    private Integer saturation;

    /**
     * 膚色明度
     */
    @JsonProperty("brightness")
    private Integer brightness;
}
